example.py

```diff
@@ -114,6 +114,14 @@ def __init__(self):
                                     {'scale_factor': 0.8}])
         saveall_button = QtWidgets.QPushButton(saveall, 'Stack, offset')
 
+        # Stack and offset icons, draw path
+        saveall_path = qta.icon('fa5.save', 'fa5.save',
+                           options=[{'scale_factor': 0.8,
+                                     'offset': (0.2, 0.2),
+                                     'color': 'gray', 'draw': 'path'},
+                                    {'scale_factor': 0.8, 'draw': 'path'}])
+        saveall_button_path = QtWidgets.QPushButton(saveall_path, 'Stack, offset, draw path')
+
```

Comment:
I think we should add this to the docs and probably update the GIF for `example.py`. Maybe adding to the docs some info regarding the new `draw` option